.circleci/config.yml

```diff
@@ -186,12 +193,34 @@ jobs:
           root: *workspace_root
           paths:
             - .aws
+  
+  notify-sentry-deploy:
+    executor: node-executor
+    environment:
+      SENTRY_ORG: $SENTRY_ORG
+      SENTRY_PROJECT: $SENTRY_PROJECT
+      SENTRY_ENVIRONMENT: production
+    steps:
+      - checkout
+      - run:
+          name: Create release and notify Sentry of deploy
+          command: |
+            curl -sL https://sentry.io/get-cli/ | bash
+            export SENTRY_RELEASE=$(sentry-cli releases propose-version)
+            sentry-cli releases new -p $SENTRY_PROJECT $SENTRY_RELEASE
+            sentry-cli releases set-commits $SENTRY_RELEASE --auto
+            sentry-cli releases files $SENTRY_RELEASE upload-sourcemaps path-to-sourcemaps-if-applicable
```

Comment:
A question here, mostly because I'm not familiar with circle ci, after the checkout step is the nextjs build diirectoy available for the source maps to be uploaded? 

Also shouldn't `path-to-sourcemaps-if-applicable` be a directoy path?